config/site_settings.yml

```diff
@@ -2126,8 +2126,13 @@ navigation:
     choices:
       - "default"
       - "unread_new"
-  enable_new_notifications_menu:
-    default: true
+  enable_custom_sidebar_sections:
```

Comment:
? `enable_custom_sidebar_sections` doesn't seem relevant to this PR - accidental diff from a merge/rebase?